README.md

```diff
@@ -319,11 +322,15 @@ To create a remote control session, follow these steps:
 
    ```bash
    $ curl -ks https://localhost:8080/session \
+          -u support:1234 \
           -F id="helping-joe" \
           -F ttl=3600 \
           -F username=admin \
           -F password=foobaz |jq
    ```
+   
+   - Basic authentication `-u` is required the credentials are part of `plexus.conf` file
+   under `PLEXUS_AUTH_USER` and `PLEXUS_AUTH_PASSWORD` keys.
```

Comment:
This sounds a little weird, how about:
```suggestion
   - The configured credentials `PLEXUS_AUTH_USER` and `PLEXUS_AUTH_PASSWORD` are required when creating a session.
```

Reply:
Yeah, I agree. Writing is not my strong side.